Makefile

```diff
@@ -1,5 +1,5 @@
 NAME = phusion/baseimage
-VERSION = 0.9.16
+VERSION = 0.9.16-bukzor0
```

Comment:
This should be defined by the maitainers, since it could go to the next or further release.